client/vscode-lib/src/controller.ts

```diff
@@ -119,75 +102,76 @@ export function createController({
         providers,
     })
 
-    const errorTapObserver: Partial<TapObserver<any>> = {
-        next(): void {
-            errorWaiter.gotError(false)
-        },
-        error(): void {
-            // Show an error notification unless we've recently shown one (to avoid annoying
-            // the user).
-            const shouldNotify = errorWaiter.timeSinceLastError() > 1000 * 60 * 15 /* 15 min */
-            if (shouldNotify) {
-                showErrorNotification(outputChannel)
-            }
-
-            errorWaiter.gotError(true)
-        },
+    const showErrorNotification = async (providerUri: string | undefined, error: any) => {
+        // TODO(keegan) try convert providerUri to a meta.name and an "Open
```

Comment:
TODO to show meta.name is error message?

Reply:
This is fixed now